README.md

```diff
@@ -61,143 +71,326 @@ $ git clone https://github.com/amazon-connect/amazon-connect-chatjs
+const { connectCalled, connectSuccess } = await chatSession.connect();
 ```
 
-Use the chatSession object to subscribe to the following callbacks. Example - The onMessage callback is used to handle any messages sent from one of the participants or the chat service.
+Wraps the [CreateParticipantConnection](https://docs.aws.amazon.com/connect-participant/latest/APIReference/API_CreateParticipantConnection.html) API.
 
+The arguments and response do not overlap with the API request or response.
+
+**Note:** If the operation fails, the `Promise` will reject, but the error will have the same schema as a successful response.
+
+#### `chatSession.getTranscript()`
+```js
+const awsSdkResponse = await chatSession.getTranscript({
+  maxResults: 100,
+  sortOrder: "ASCENDING"
+});
+const { InitialContactId, NextToken, Transcript } = awsSdkResponse.data;
 ```
-chatSession.onConnectionBroken(data => {console.log("connection broken with server")});
-chatSession.onTyping(data => {console.log("someone is typing! details:", data)});
-chatSession.onMessage(data => {console.log("there  is message! details:", data)});
-chatSession.onConnectionEstablished(data => {console.log("connection established with server")});
-chatSession.onEnded(() => {console.log("chat has ended")})
+
+Wraps the [GetTranscript](https://docs.aws.amazon.com/connect-participant/latest/APIReference/API_GetTranscript.html) API.
+
+The arguments are based on the [API request body](https://docs.aws.amazon.com/connect-participant/latest/APIReference/API_GetTranscript.html#API_GetTranscript_RequestSyntax) with the following differences:
+
+- Fields are in `camelCase`.
+- `MaxResults` defaults to `15`.
+- `ScanDirection` defaults to `BACKWARD` always.
+- `SortOrder` defaults to `ASCENDING`.
+
+The response `data` is the same as the [API response body](https://docs.aws.amazon.com/connect-participant/latest/APIReference/API_GetTranscript.html#API_GetTranscript_ResponseSyntax).
+
+#### `chatSession.sendEvent()`
+```js
+const awsSdkResponse = await chatSession.sendEvent({
+  contentType: "application/vnd.amazonaws.connect.event.typing"
+});
+const { AbsoluteTime, Id } = awsSdkResponse.data;
 ```
-# Usage:
 
-### `connect.ChatSession.connect`
-`Method param:` args
+Wraps the [SendEvent](https://docs.aws.amazon.com/connect-participant/latest/APIReference/API_SendEvent.html) API.
+
+The arguments are based on the [API request body](https://docs.aws.amazon.com/connect-participant/latest/APIReference/API_SendEvent.html#API_SendEvent_RequestSyntax) with the following differences:
+
+- Fields are in `camelCase`.
+- `ClientToken` cannot be specified.
+- `ContentType` allows the following values:
+  - `"application/vnd.amazonaws.connect.event.chat.ended"`
+  - `"application/vnd.amazonaws.connect.event.participant.joined"`
+  - `"application/vnd.amazonaws.connect.event.participant.left"`
+  - `"application/vnd.amazonaws.connect.event.transfer.succeeded"`
```

Comment:
The previous documentation incorrectly referred to this event as `application/vnd.amazonaws.connect.event.transfer.succeed` (notice `succeeded` vs `succeed`).